examples/platform/beken/common/CommonDeviceCallbacks.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include "CommonDeviceCallbacks.h"
+
+#include <app-common/zap-generated/attribute-id.h>
+#include <app-common/zap-generated/cluster-id.h>
+#include <app/server/Dnssd.h>
+#include <app/util/util.h>
+#include <lib/support/CodeUtils.h>
+
+using namespace chip;
+using namespace chip::DeviceLayer;
+using namespace chip::System;
+
+// DeviceCallbacksDelegate * appDelegate = nullptr;
+
+void CommonDeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, intptr_t arg)
+{
+    ChipLogProgress(Zcl, "DeviceEventCallback, event->Type:%d \r\n", event->Type);
+    switch (event->Type)
+    {
+    case DeviceEventType::kInternetConnectivityChange:
+        OnInternetConnectivityChange(event);
+        break;
+
+    // case DeviceEventType::kSessionEstablished:
```

Comment:
same here: delete instead of commenting out.

Reply:
OK. We will remove it.